### PR TITLE
adding variable propagation for notification-email

### LINKED ISSUE #1073: notification-email set in en-alerting module does not update
### TL;DR
can't set notification-email variable if importing the alerting terraform as a module

**Expected behavior**
notification-email reflects in module

**Observed behavior**
variable is not

### Reproduction
1. Import this repo (/terraform/alerting) as a module in tf
2. Set notification-email variable in the module
3. Terraform plan will not update the email config

### PR DESCRIPTION
**Pull request by bschlaman**
Fixes #1073

### CODE REVIEW

**Comment by bschlaman** [comment]
@googlebot I signed it!

File: terraform/alerting/main.tf [hunk 1]
```diff
@@ -7,6 +7,7 @@ module "verification-server" {
   apiserver_hosts = var.apiserver_hosts
   adminapi_hosts  = var.adminapi_hosts
   server_hosts    = var.server_hosts
+	notification-email = var.notification-email
```
Comment (whaught) [inline terraform/alerting/main.tf]:
```suggestion
	notification-email = var.notification-email
```
  notification-email = var.notification-email

Comment (whaught) [reply terraform/alerting/main.tf]: I can't tell if that suggestion actually worked, but the whitespace looks off

**Review by whaught — APPROVED** [review]

**Pull request merged**